### PR TITLE
[fix] Fix issue of text and images being squeezed on mobile screens.

### PR DESCRIPTION
**Pull request by erik1110**
## Types of changes
Fix issue of text and images being squeezed on mobile screens.

* **Bugfix**

## Description
1. It will squeezed on mobile screens
- Before
image
- After
image
2. The image will not show properly because of margin setting.
- Before
image
- After
image

### CODE REVIEW

**Review by josix — CHANGES_REQUESTED** [review]
Thanks for the update @erik1110, the whole layout looks nice! Would you like to help us refactor the css coding convention that adopts tailwindcss utilities inline style like #395, thanks!

File: pages/about/code-of-conduct.vue [hunk 1]
```diff
@@ -272,6 +278,8 @@ export default {
     }
     .sub2_img {
         @apply flex;
+        margin-left: -1.5rem;
+        margin-right: -1.5rem;
```
Comment (yungshenglu) [inline pages/about/code-of-conduct.vue:282]: Maybe we can write into `@apply` like this:
```
@apply my-[-1.5rem]
```

File: pages/about/code-of-conduct.vue [hunk 2]
```diff
@@ -258,6 +258,12 @@ export default {
         @apply mt-20;
     }
 }
+@media (max-width: 500px) {
+    .img_text {
+        font-size: 0.4rem;
+        white-space: normal;
```
Comment (yungshenglu) [inline pages/about/code-of-conduct.vue:264]: Please use Tailwind syntax with `@apply` for these styles.

Comment (erik1110) [reply pages/about/code-of-conduct.vue]: OK

File: pages/about/code-of-conduct.vue (hunk 1 above)
Comment (erik1110) [reply pages/about/code-of-conduct.vue:279]: OK

File: pages/about/code-of-conduct.vue [hunk 3]
```diff
@@ -241,23 +241,26 @@ export default {
...
 .img_text {
-    @apply absolute mt-24 whitespace-nowrap text-center;
-    font-size: 0.8rem;
+    @apply absolute mt-24 whitespace-nowrap text-center text-sm;
 }
 @media (max-width: 640px) {
     .img_text {
         @apply mt-20;
     }
 }
```
Comment (baby230211) [inline pages/about/code-of-conduct.vue:258]: could you help to put the following scope on the class with
`sm:mt-20`

File: pages/about/code-of-conduct.vue [hunk 4]
```diff
@@ -241,23 +241,26 @@ export default {
...
+@media (max-width: 500px) {
+    .img_text {
+        @apply text-xs whitespace-normal;
+    }
+}
```
Comment (baby230211) [inline pages/about/code-of-conduct.vue:263]: may i ask why 500px ?
is there any cases you have to handle with 500px?
the below code is tailwind rule on responsive screen
```
screens: {
            xs: '360px',
            sm: '640px',
            md: '768px',
            lg: '1194px',
            xl: '1440px',
            '2xl': '1600px',
        },
```

**Review by josix — APPROVED** [review]
LGTM

**Comment by josix** [comment]
Hello @erik1110, I found that there are some lint errors in the original code, I have applied`npm run prettier:lint` to fix it, please take a look!

**Pull request merged**